aiconfig-docs/docs/gradio-notebook.md

```diff
@@ -32,6 +32,10 @@ You can:
   - [app.py](https://huggingface.co/spaces/lastmileai/gradio-notebook-template/blob/main/app.py)
   - [requirements.txt](https://huggingface.co/spaces/lastmileai/gradio-notebook-template/blob/main/requirements.txt)
 
+:::caution
+Please ensure the `sdk_version` in your Space's `README.md` is set to `sdk_version: 4.16.0` or lower due to compatibilty issues in higher `gradio` package versions
```

Comment:
Let's add a URL from our quickstart so people can easily use this as reference: https://huggingface.co/spaces/lastmileai/gradio-notebook-quickstart/blob/main/README.md

Reply:
Going to use https://huggingface.co/spaces/lastmileai/gradio-notebook-template/blob/main/README.md since we may be removing quickstart

Reply:
![Screenshot 2024-02-13 at 11 53 04 PM](https://github.com/lastmile-ai/aiconfig/assets/5060851/e20bd8dc-688b-46e6-9e79-4b8ddfb61e3a)